tests/unit/s2n_tls13_cert_verify_test.c

```diff
@@ -46,18 +47,29 @@ int run_tests(const struct s2n_tls13_cert_verify_test *test_case, s2n_mode verif
     const char *key_file = test_case->key_file;
     struct s2n_signature_scheme sig_scheme = *test_case->sig_scheme;
 
-    struct s2n_config *config = NULL;
-    EXPECT_NOT_NULL(config = s2n_config_new());
+    DEFER_CLEANUP(struct s2n_config *config = s2n_config_new(), s2n_config_ptr_free);
+    EXPECT_NOT_NULL(config);
     EXPECT_SUCCESS(s2n_config_set_cipher_preferences(config, "20200207"));
 
+    DEFER_CLEANUP(struct s2n_cert_chain_and_key *cert_chain = s2n_cert_chain_and_key_new(), 
+            s2n_cert_chain_and_key_ptr_free);
+    EXPECT_NOT_NULL(cert_chain);
+
+    char cert_chain_pem[S2N_MAX_TEST_PEM_SIZE] = { 0 };
+    char private_key_pem[S2N_MAX_TEST_PEM_SIZE] = { 0 };
+
+    EXPECT_SUCCESS(s2n_read_test_pem(cert_file, &cert_chain_pem[0], S2N_MAX_TEST_PEM_SIZE));
+    EXPECT_SUCCESS(s2n_read_test_pem(key_file, &private_key_pem[0], S2N_MAX_TEST_PEM_SIZE));
+    EXPECT_SUCCESS(s2n_cert_chain_and_key_load_pem(cert_chain, cert_chain_pem, private_key_pem));
```

Comment:
Can't you use the newer [s2n_test_cert_chain_and_key_new](https://github.com/aws/s2n-tls/blob/main/tests/testlib/s2n_testlib.h#L200-L201)? Or is there something different about how certs need to be loaded here?

Reply:
The raw PEM is used in the test cases, so we want to keep access to the buffers: https://github.com/aws/s2n-tls/blob/9fb55ec5944f1293680678b016a99824d16508bb/tests/unit/s2n_tls13_cert_verify_test.c#L93-L95